src/Sentry.AspNet/SentryHttpServerUtilityExtensions.cs

```diff
@@ -0,0 +1,26 @@
+using System.Web;
+using Sentry.Protocol;
+
+namespace Sentry.AspNet;
+/// <summary>
+/// HttpServerUtility extensions.
+/// </summary>
+public static class SentryHttpServerUtilityExtensions
+{
+    /// <summary>
+    /// Caoture the last error from the given HttpServerUtility and send to Sentry.
+    /// </summary>
+    /// <param name="server">The HttpServerUtility that contains the last error.</param>
+    /// <param name="hub">(optional) The Hub that will capture the exception.</param>
+    /// <returns>A SentryId.</returns>
+    public static SentryId CaptureLastError(this HttpServerUtility server, IHub? hub = null)
```

Comment:
Can we have an internal overload for testing instead of this default paramter?

```suggestion
    public static SentryId CaptureLastError(this HttpServerUtility server) => server.CaptureLastError(HubAdapter.Instance);
    
    // for testing
    internal static SentryId CaptureLastError(this HttpServerUtility server, IHub hub)
```

Reply:
Absolutely, thanks for pointing that out.